src/rpc/mempool.cpp

```diff
@@ -143,19 +149,30 @@ static RPCHelpMan testmempoolaccept()
                                    "Array must contain between 1 and " + ToString(MAX_PACKAGE_COUNT) + " transactions.");
             }
 
-            const CFeeRate max_raw_tx_fee_rate = request.params[1].isNull() ?
-                                                     DEFAULT_MAX_RAW_TX_FEE_RATE :
-                                                     CFeeRate(AmountFromValue(request.params[1]));
+            CFeeRate max_raw_tx_fee_rate = DEFAULT_MAX_RAW_TX_FEE_RATE;
+
+            if (!request.params[1].isNull()) {
+                const UniValue& options = request.params[1];
+                RPCTypeCheckObj(options,
```

Comment:
Should have backward compatibility for now

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/25577/commits/618a7920d1bcbf25c67ed15b5f8c43afbcbbf323.